lib/util/git/private-key.spec.ts

```diff
@@ -1,4 +1,7 @@
-import { mocked } from '../../../test/util';
+import fs from 'fs-extra';
+import { any, mockReset } from 'jest-mock-extended';
+import { Fixtures } from '../../../test/fixtures';
+import { mockedDeep } from '../../../test/util';
```

Comment:
Moving this import from the top to here is necessary to satisfy ESLint, otherwise it raises the following error:

```
Error: `../../../test/util` import should occur after import of `../../../test/fixtures`
```